test/common/index.js

```diff
@@ -55,6 +54,8 @@ exports.isLinuxPPCBE = (process.platform === 'linux') &&
                        (os.endianness() === 'BE');
 exports.isSunOS = process.platform === 'sunos';
 exports.isFreeBSD = process.platform === 'freebsd';
+exports.isOpenBSD = process.platform === 'openbsd';
+exports.execPathOpenBSD = path.join(__dirname, '../../', process.execPath);
```

Comment:
What do you about exporting `execPath` like this

```js
exports.execPath = exports.isOpenBSD
  ? path.join(__dirname, '../../', process.execPath)
  : process.execPath;
```

and use `common.execPath` instead of `process.execPath` in the tests? This should slightly reduce the diff noise.